server/index.js

```diff
@@ -1,42 +1,47 @@
-import http from 'http'
 import { resolve, join } from 'path'
 import { parse } from 'url'
+import http from 'http'
 import send from 'send'
+import accepts from 'accepts'
+import {
+  renderToHTML,
+  renderErrorToHTML,
+  renderJSON,
+  renderErrorJSON,
+  sendHTML
+} from './render'
 import Router from './router'
-import { render, renderJSON, errorToJSON } from './render'
 import HotReloader from './hot-reloader'
 import { resolveFromList } from './resolve'
 
 export default class Server {
-  constructor ({ dir = '.', dev = false, hotReload = false }) {
```

Comment:
`hotReload` option is removed since only `dev: true, hotReload: true` or `dev: false, hotReload: false` work for now anyway.